pkg/autodiscovery/autoconfig.go

```diff
@@ -7,44 +7,24 @@ package autodiscovery
 
 import (
 	"context"
-	"expvar"
-	"fmt"
 	"sync"
 	"sync/atomic"
 	"time"
 
-	"github.com/DataDog/datadog-agent/pkg/autodiscovery/configresolver"
 	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
 	"github.com/DataDog/datadog-agent/pkg/autodiscovery/listeners"
 	"github.com/DataDog/datadog-agent/pkg/autodiscovery/providers"
 	"github.com/DataDog/datadog-agent/pkg/autodiscovery/scheduler"
 	"github.com/DataDog/datadog-agent/pkg/collector/check"
 	"github.com/DataDog/datadog-agent/pkg/config"
-	"github.com/DataDog/datadog-agent/pkg/secrets"
 	"github.com/DataDog/datadog-agent/pkg/status/health"
 	"github.com/DataDog/datadog-agent/pkg/tagger"
 	"github.com/DataDog/datadog-agent/pkg/util/containers"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 	"github.com/DataDog/datadog-agent/pkg/util/retry"
 )
 
-var (
-	listenerCandidateIntl = 30 * time.Second
-	acErrors              *expvar.Map
-	errorStats            = newAcErrorStats()
-)
-
-var secretsDecrypt = secrets.Decrypt
-
-func init() {
-	acErrors = expvar.NewMap("autoconfig")
-	acErrors.Set("ConfigErrors", expvar.Func(func() interface{} {
-		return errorStats.getConfigErrors()
-	}))
-	acErrors.Set("ResolveWarnings", expvar.Func(func() interface{} {
-		return errorStats.getResolveWarnings()
-	}))
-}
+var listenerCandidateIntl = 30 * time.Second
```

Comment:
nit: `const`

Reply:
This is changed in a test (and, just moved here, not added)